.config

```diff
@@ -3,3 +3,4 @@ repo-owner = fluture-js
 repo-name = fluture-node
 source-files = index.mjs
 opening-delimiter = ```js
+module-type = esm
```

Comment:
A concept option we might add to `sanctuary-scripts`.

Reply:
Yes, let's do this! Would you like to submit a pull request?

Reply:
https://github.com/sanctuary-js/sanctuary-scripts/pull/26